dev/cnf/oss_dependencies.maven

```diff
@@ -37,6 +37,7 @@ com.ibm.async:asyncutil:0.1.0
 com.ibm.db2:jcc:11.1.4.4
 com.ibm.ws.componenttest:mantis-collections:2.5.0
 com.ibm.ws.componenttest:mantis-nls-standalone:2.5.0
+com.ibm.ws.io.opentracing:com-ibm-ws-io-opentracing-zipkintracer:0.33
```

Comment:
As this artifact is not on maven central, this line should be in `oss_ibm.maven` and the artifact uploaded to DHE.